wp-job-manager-functions.php

```diff
@@ -1763,3 +1763,14 @@ function job_manager_renew_job_listing( $job ) {
 
 	wp_update_post( $update_job );
 }
+
+/**
+ * Checks if the wc paid listings has the minimum version when installed and activated.
+ *
+ * @since $$next-version$$
+ *
+ * @return bool
+ */
+function job_manager_is_wcpl_renew_compatible() {
+	return ! class_exists( 'WC_Paid_Listings' ) || version_compare( JOB_MANAGER_WCPL_VERSION, '2.9.9', '>' );
```

Comment:
This disables the feature, to test, change the version number to something less eg `2.9.8`